content/plugins/aws.md

```diff
@@ -152,26 +189,222 @@ See the [common Runtime Properties section](#runtime-properties).
 Note that the actual IP is available via the `aws_resource_id` runtime-property.
 
 
+## cloudify.aws.nodes.VPC
```

Comment:
I'd either provide a link to an explanation on VPC or provide a one-sentence explanation here.